uint32 complete_id

---

#Enumerated action types
uint8 STEP = 0
uint8 VIEW = 1
uint8 SLEEP = 2

#major id is for chronological ordering later
uint32 major_id

#action type tells which subplanner should handle this move
uint8 action_type

#which frame to move/target, and where to go
string node_name
float32 x
float32 y

#how long to sleep for
duration sleep